app/views/bgeigie_imports/_head.html.erb

```diff
@@ -13,6 +13,14 @@
           <%- end -%>
         </li>
       <%- end -%>
+      <%- if !@bgeigie_import.awaiting_approval? && user_signed_in? && current_user.moderator? && !@bgeigie_import.done? -%>
```

Comment:
I would say - show it always and remove any intelligence from when to show it - it’s useless after approval for sure

Pieter

> On Apr 16, 2016, at 11:36 AM, Mat Schaffer <notifications@github.com> wrote:
> 
> In app/views/bgeigie_imports/_head.html.erb <https://github.com/Safecast/safecastapi/pull/162#discussion_r59961003>:
> 
> > @@ -13,6 +13,14 @@
> >            <%- end -%>
> >          </li>
> >        <%- end -%>
> > +      <%- if !@bgeigie_import.awaiting_approval? && user_signed_in? && current_user.moderator? && !@bgeigie_import.done? -%>
> @seanbonner <https://github.com/seanbonner> definitely set to show only if drive is not awaiting approval (presumably approved). Should the condition be just visible to any moderator?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/Safecast/safecastapi/pull/162/files/05ce4ee42c3083e2e6c1264423e973f661a0f72e#r59961003>